queries/gdscript/highlights.scm

```diff
@@ -1,38 +1,44 @@
 ;; Basic
+(ERROR) @error
+
 (identifier) @variable
 (name) @variable
 (type) @type
-(comment) @comment
-(get_node) @string
+(comment) @comment @spell
+(string_name) @string
```

Comment:
string_name is a confusing name, can you provide more context/info as to what it is?

Reply:
Sure, it's syntax sugar for Godot's concept of interned strings. They look like:

```gdscript
&"interned string"
```